uq.esys.escript.appdata.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+
+  <!-- Copyright 2019 Adam Ellery <a.ellery@uq.edu.au> -->
```

Comment:
The copyright should be before the component tag.  You can run appstream-util validate on it. Also, it would be better if you can upstream those files the appdata, the desktop files, and the icons. They are not Flathub/Flatpak specific standards.